spec/controllers/csv_files_controller_spec.rb

```diff
@@ -0,0 +1,265 @@
+require 'rails_helper'
+
+require 'support/controller_macros'
+require 'support/devise'
+require 'support/shared_examples_for_authentication'
+
+RSpec.describe CsvFilesController, type: :controller do
+	#############################################################################
```

Comment:
hmm...looks like your editor might have tabs enabled some of the time?  There's some indentation inconsistency here.